mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -142,7 +142,7 @@ properties:
                   where HH : [00-23] and MM : [00] GMT. eg: 21:00
                 required: true
                 validation: !ruby/object:Provider::Terraform::Validation
-                  function: 'verify.ValidateHourlyOnly'
+                  regex: '^([01][0-9]|2[0-3]):00$'
```

Comment:
Note for posterity:

I tested the regex and can see that these cases are all identified as invalid

- 0:00
- 0:01
- 1:00
- 10:10
- 24:00

and these are identified as valid:

- 01:00
- 10:00
- 20:00
- 23:00